update.sh

```diff
@@ -12,14 +12,19 @@ while [ -h "$SOURCE" ]; do
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
+# Fetch the code base
+mkdir -p $GOPATH/src/github.com/theupdateframework
+cd $GOPATH/src/github.com/theupdateframework/
+git clone https://github.com/theupdateframework/notary
+
```

Comment:
You can't just clone the master branch here. The current assumption is that this is manually built from the right checkout.

Reply:
So this means that the user should manually checkout ```https://github.com/theupdateframework/notary``` and then build ```notary``` code?

Actually, I tried to automate this by fetching all the dependencies for the user who wants to generate binaries for different architectures.

Reply:
Yeah that really is not helpful for this repo, that workflow is not going to work, and if it did it would need all the architectures committed as different binaries. The proposal in https://github.com/docker-library/official-images/pull/4718 is that we do release tarballs for more architectures, and use those for building official images.